src/Nancy/Conventions/BuiltInCultureConventions.cs

```diff
@@ -133,7 +133,12 @@ public static CultureInfo CookieCulture(NancyContext context, GlobalizationConfi
         /// <returns>CultureInfo from CurrentThread</returns>
         public static CultureInfo ThreadCulture(NancyContext context, GlobalizationConfiguration configuration)
         {
+#if DOTNET5_4
+            CultureInfo.CurrentCulture = new CultureInfo(configuration.SupportedCultureNames.First());

```

Comment:
Here's the docs for it

CurrentCulture

Gets or sets the CultureInfo
<http://dotnet.github.io/api/System.Globalization.CultureInfo.html#System_Globalization_CultureInfo>
object
that represents the culture used by the current thread.


Don't forget to click the link to MS way of doing it

On Friday, 19 February 2016, Andreas Håkansson <notifications@github.com>
wrote:

> In src/Nancy/Conventions/BuiltInCultureConventions.cs
> <https://github.com/NancyFx/Nancy/pull/2290#discussion_r53522465>:
>
> > @@ -133,7 +133,12 @@ public static CultureInfo CookieCulture(NancyContext context, GlobalizationConfi
> >          /// <returns>CultureInfo from CurrentThread</returns>
> >          public static CultureInfo ThreadCulture(NancyContext context, GlobalizationConfiguration configuration)
> >          {
> > +#if DOTNET5_4
> > +            CultureInfo.CurrentCulture = new CultureInfo(configuration.SupportedCultureNames.First());
>
> Is this even the same thing as Thread.CurrentThread.CurrentCulture maybe
> it is the ThreadCulture that should not be supported on dotnet5_4 at all?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/2290/files#r53522465>.
>